Keas.Mvc/Controllers/EquipmentController.cs

```diff
@@ -98,6 +109,7 @@ public string GetTeam()
                 _context.EquipmentAssignments.Remove(eq.Assignment);
                 eq.Assignment = null;
                 await _context.SaveChangesAsync();
+                await _eventService.TrackUnAssignEquipment(equipment, await _securityService.GetUser());
```

Comment:
@jcubbage why not have the event service call the security service itself?  Seems like it would save a lot of typing and extra service instantiations

Reply:
@jcubbage cool, i'd say give it a quick try and see if it's working now that credentials are being passed, and if no don't worry about it